File: gamsojji/backjoon/step6/2941.js
let fs = require("fs");
const filePath = process.platform === 'linux' ? "/dev/stdin" : `${__dirname}/input.txt`;
let input = fs.readFileSync(filePath).toString().trim();
let cb = ["c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z="];

cb.forEach((b, idx) => {
    input = input.split(b).join("t");
})

console.log(input.length);

File: rolemadelen/baekjoon/step6/2941.rs
use std::io::stdin;

fn main() {
    let mut input = String::new();
    stdin().read_line(&mut input).unwrap();
    let input = input.trim();

    let str = input
                .replace("dz=", "d")
                .replace("d-", "d")
                .replace("c=", "c")
                .replace("c-", "c")
                .replace("lj", "l")
                .replace("nj", "n")
                .replace("s=", "s")
                .replace("z=", "z");

    println!("{}", str.len());
}
